src/schema/README.md

```diff
@@ -308,38 +331,41 @@ check will fail.
 These files **do not** describe how objects of different types
-(for example file suffixes and file entities) interact with one another, or
-whether objects are required in a given dataset or file.
+(for example file suffixes and file entities) interact with one another,
+or whether objects are required in a given dataset or file.
 
 ### Overview
 
 There are currently 11 sub-namespaces, which fall into five rough categories.
+
 The namespaces are:
 
 | Namespace                   | Description                                                                         | Group            |
 | --------------------------- | ----------------------------------------------------------------------------------- | ---------------- |
-| `objects.common_principles` | Terms that are used throughout BIDS                                                 | General terms    |
-| `objects.modalities`        | Broad categories of data represented in BIDS, roughly matching recording instrument | General terms    |
-| `objects.entities`          | Name-value pairs appearing in filenames                                             | Name/value terms |
-| `objects.metadata`          | Name-value pairs appearing in JSON files                                            | Name/value terms |
 | `objects.columns`           | Column headings and values appearing in TSV files                                   | Name/value terms |
+| `objects.common_principles` | Terms that are used throughout BIDS                                                 | General terms    |
 | `objects.datatypes`         | Subdirectories that organize files by type (such as `anat`, `eeg`)                  | Value terms      |
-| `objects.suffixes`          | Filename suffixes that describe the contents of the file                            | Value terms      |
+| `objects.entities`          | Name-value pairs appearing in filenames                                             | Name/value terms |
 | `objects.extensions`        | Filename component that describe the format of the file                             | Value terms      |
-| `objects.formats`           | Terms that define the forms values (for example, in metadata) might take            | Formats          |
 | `objects.files`             | Files and directories that may appear at the root of a dataset                      | Files            |
+| `objects.formats`           | Terms that define the forms values (for example, in metadata) might take            | Formats          |
+| `objects.metadata`          | Name-value pairs appearing in JSON files                                            | Name/value terms |
+| `objects.modalities`        | Broad categories of data represented in BIDS, roughly matching recording instrument | General terms    |
+| `objects.suffixes`          | Filename suffixes that describe the contents of the file                            | Value terms      |
```

Comment:
These were grouped by the `Group` column, and the groups were ordered in the order they are explained below, which was determined by which group will cover the most ground while introducing the fewest new concepts.

Reply:
ah I totally missed that

sorry

will revert this: cannot wait to have frameworks for doc that allow for sorting of markdown / rst tables by any column...